aikau/src/main/resources/alfresco/renderers/_ActionsMixin.js

```diff
@@ -219,12 +219,17 @@ define(["dojo/_base/declare",
          array.forEach(this.widgetsForActions, function(action) {
             if (action && action.name)
             {
-
+               // It's expected that an Actions rendererer will be used inside a row of Document List view. The
+               // row itself may or may note have its own scope and where some actions will require that the
+               // list is refreshed (e.g. when items are deleted, moved or copied) in which case the responseScope
+               // should be addressed at the list. We therefore need to set the response scope accordingly...
+               var responseScope = this.publishToParent ? this.parentPubSubScope : this.pubSubScope;
```

Comment:
I'm sure it's already been considered, but just to double-check, the response scope definitely won't be correctly generated automatically when the alfPublish is called?